packages/core/src/codewhisperer/activation.ts

```diff
@@ -293,7 +298,8 @@ export async function activate(context: ExtContext): Promise<void> {
 
     if (auth.isConnectionExpired()) {
         auth.showReauthenticatePrompt().catch(e => {
-            getLogger().error('showReauthenticatePrompt failed: %s', (e as Error).message)
+            const defaulMsg = localize('AWS.generic.message.error', 'Failed to reauth:')
+            void logAndShowError(localize, e, 'showReauthenticatePrompt', defaulMsg)
```

Comment:
If `showReauthenticatePrompt` failed, no message was shown here. Fixed now, but need to revisit this. Not great to special-case it here.